File: src/main/java/econtroller/modeler/ModelPort.java
package econtroller.modeler;

import cloud.elb.SenseData;
import se.sics.kompics.PortType;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-05-08
 *
 */

public class ModelPort extends PortType {
	{
		negative(StartModeler.class);
        negative(SenseData.class);
	}
}

File: src/main/java/econtroller/modeler/ModelPort.scala
package econtroller.modeler

import se.sics.kompics.PortType
import cloud.elb.SenseData

/**
 * @author Amir Moulavi
 * @date 2011-07-24
 *
 */
class ModelPort extends PortType {

  negative(classOf[StartModeler])
  negative(classOf[SenseData])

}